Functions.Templates/Templates/SqlInputBinding-JavaScript/function.json

```diff
@@ -0,0 +1,21 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "name": "req",
+      "type": "httpTrigger",
+      "direction": "in",
+      "methods": [
+        "get"
+      ]
+    },
+    {
+      "type": "sql",
+      "name": "results",
+      "direction": "in",
+      "commandText": "select * from [dbo].[table1]",
+      "commandType": "Text",
+      "connectionStringSetting": "SqlConnectionString"
+    }
```

Comment:
This should also have the out response 

```
    {
      "type": "http",
      "direction": "out",
      "name": "res"
    }
```

Reply:
for input binding ?

Reply:
Yes. The response is for the HTTP trigger